.github/workflows/ci_cd.yml

```
@@ -188,7 +188,7 @@ jobs:
         working-directory: pyvista
 
       - name: Unit Testing
-        run: xvfb-run python -m pytest -v --allow_useless_fixture --generated_image_dir gen_dir tests/plotting/test_plotting.py
+        run: xvfb-run python -m pytest -v --allow_useless_fixture --generated_image_dir gen_dir tests/plotting/test_plotting.py -n2
```

Should we run _all_ plotting tests? That way, we can enable `disallow_unused_cache` and do a full parallel test run. Otherwise, we don't have coverage for this combination.
```suggestion
        run: xvfb-run python -m pytest -v --allow_useless_fixture --generated_image_dir gen_dir tests/plotting --disallow_unused_cache -n2
```

---

Agreed, although it would be nice to run a sub-selection of the tests since there are so many. We can't really do that with the way we've written `disallow_unused_cache`.